src/JD.Efcpt.Build.Templates/templates/efcptbuild/EfcptProject.csproj

```diff
@@ -0,0 +1,61 @@
+<!--
+  JD.Efcpt.Sdk Project Template
+  
+  This project uses JD.Efcpt.Sdk to automatically generate EF Core models from a database project.
+  
+  The SDK:
+  - Extends Microsoft.NET.Sdk with EF Core Power Tools integration
+  - Automatically detects the SQL project via ProjectReference
+  - Builds the SQL project to DACPAC and generates EF Core models
+  
+  No JD.Efcpt.Build PackageReference needed - the SDK handles everything!
+-->
+<Project Sdk="JD.Efcpt.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Why not net10.0 ?

Reply:
Make sure not to confuse EF Core version and .NET version.

EF Core 8 - .NET 8,9,10
EF Core 9 - .NET 8,9,10
EF Core 10 - .NET 10